src/scs/slsa-framework.md

```diff
@@ -0,0 +1,22 @@
+# SLSA Framework
+
+Supply chain Levels for Software Artifacts (SLSA) is a security framework for tampering prevention, integrity improvement and securing packages and infrastructure of a project.
+
+## Levels of assurance
+
+One of the requirements for the solution is to reach SLSA Level 4 and even go beyond that. This requires a lot of process changes as well as technical work. Current status for Thincompute project is visualized at the separate wiki page: Supply Chain Security 
+
+The SLSA model consists of 4 levels, offering incremental level of anti-tampering protection. Levels 1-3 represent milestones with certain integrity guarantees, whereas level 4 is an ideal end state.
+
+**Level 0** means no SLSA compliance and no guarantees are given.
+
+**Level 1** demands fully scripted/automated build process and provenance generation. It offers basic confidence of the origin of the software but doesn’t provide any tamper resistance.
```

Comment:
...demands **a** fully scripted/automated...

... confidence **in** the origin...